lib/rubocop/cop/rspec/shared_examples_style.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Enforces use of string to titleize shared examples.
+      #
+      # @example
+      #   # bad
+      #   it_behaves_like :foo_bar_baz
+      #   it_should_behave_like :foo_bar_baz
+      #   shared_examples :foo_bar_baz
+      #   shared_examples_for :foo_bar_baz
+      #   include_examples :foo_bar_baz
+      #
+      #   # good
+      #   it_behaves_like 'foo bar baz'
+      #   it_should_behave_like 'foo bar baz'
+      #   shared_examples 'foo bar baz'
+      #   shared_examples_for 'foo bar baz'
+      #   include_examples 'foo bar baz'
+      #
+      class SharedExamplesStyle < Cop
+        def_node_matcher :shared_examples_style, <<-PATTERN
+          (send nil? {#{(SharedGroups::ALL + Includes::ALL).node_pattern}} $sym)
```

Comment:
We want to capture everything that isn’t a string, right? So I think it should be `$!str` instead of `$sym`.